openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/OpenEdgeComponents.java

```diff
@@ -144,7 +148,7 @@ private void registerLicences(LicenceRegistrar[] licRegistrars, String permanent
     }
   }
 
-  public void initializeChecks(SensorContext context) {
+  public void initializeChecks(SensorContext context, String serverId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "serverId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1172)